workflow-casava/src/main/java/ca/on/oicr/pde/workflows/WorkflowClient.java

```diff
@@ -338,4 +338,11 @@ protected SqwFile createOutputFile(String workingPath, String outputFileName, St
         return file;
     }
 
+    // TODO remove me after sonarcloud does its thing
+    public void junkMethodIsJunk(String unusedParam) throws Exception {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "unusedParam". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)